packages/trade-widget/truffle.js

```diff
@@ -0,0 +1,10 @@
+module.exports = {
+  contracts_build_directory: "../contracts/artifacts",
+  networks: {
+    development: {
+      host: "127.0.0.1",
+      port: 8545,
+      network_id: "*" // Match any network id
+    }
+  }
+};
```

Comment:
Eeeek... Are you using Truffle?

Reply:
Oh no this is from experimenting with trying to get the contract working with truffles new debugger, this is an unused file :P